### DISCUSSION TITLE
Unable to upload images with higher resolution

### DISCUSSION THREAD

**Discussion by sambitraze** [directus/directus#12144] [Archive]
I was trying to upload an image of 2 MB and its resolution was 1774 X 2476. How can i either disable the entire limit or is there any other way around the same

**Comment by rijkvanzanten** [comment]
How were you prevented from uploading? There's no limits that should hit those specific values you mentioned. Do you have any other proxy web server in between that might need to be configured? I believe Apache has a 2MB limit by default for example

**Reply by sambitraze** [reply]
![image](https://user-images.githubusercontent.com/23294656/158185549-c25e36fb-4b19-4923-b2c4-62821ef7d2eb.png)

**Reply by rijkvanzanten** [reply]
Do you have any other proxy web server in between that might need to be configured? I believe Apache has a 2MB limit by default for example.

**Reply by sambitraze** [reply]
i am using nginx

**Reply by rijkvanzanten** [reply]
OK. From [a quick google search](https://www.google.com/search?q=nginx+default+upload+limit), it looks like Nginx has a default max upload size of 1MB. Try configuring `client_max_body_size` in your Nginx config 👍🏻
